ios/Podfile.lock

```diff
@@ -173,14 +178,15 @@ SPEC CHECKSUMS:
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
   BVLinearGradient: 4b3f3bdf1ac1a52a5ef5e65c9585d793a7c3f825
   Crashlytics: ccaac42660eb9351b9960c0d66106b0bcf99f4fa
-  DoubleConversion: 2ca14703beeb6b302b80c2e322b9a17934d105b5
+  DoubleConversion: bb338842f62ab1d708ceb63ec3d999f0f3d98ecd
```

Comment:
Probably the changes to `DoubleConversion`, `Folly` and `glog` are not intentional in this PR. Probably they will disappear if you run `bundle exec pod install --project-directory=ios` (or simply `yarn`, since this command is part of post-install script) one more time. Because of the way cocoapods-fix-react-native works you usually need to run `pod install` twice when you add new dependencies (it warns on the first run and succeeds after that), so ensure that pod installations completes without any warnings.